Description: Enable uGreen.
 uGreen is an implementation of green threads on top of the uWSGI async
 platform.
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Last-Update: 2010-09-01

Index: uwsgi-0.9.6/uwsgiconfig.py
===================================================================
--- uwsgi-0.9.6.orig/uwsgiconfig.py	2010-09-01 12:32:10.000000000 +0300
+++ uwsgi-0.9.6/uwsgiconfig.py	2010-09-01 12:32:39.000000000 +0300
@@ -17,7 +17,7 @@
 PASTE=True
 MINTERPRETERS=True
 ASYNC=True
-UGREEN=False
+UGREEN=True
 HTTP=True
 EVDIS=False
 LDAP=True
